routes/api/account/index.js

```diff
@@ -121,12 +119,8 @@ router.put('/username', authorization.needed(), (req, res, next) => {
     .then(() => {
       res.status(204).end();
     })
-    .catch(error => {
-      if (error.code === 11000) {
-        next(errors.ErrUsernameTaken);
-      } else {
-        next(error);
-      }
+    .catch((err) => {
+      next(err);
```

Comment:
Where are we managing the type of error now? Does this work to return the right error in FE?